dev/io.openliberty.checkpoint_fat/publish/servers/xmlBindingServer/bootstrap.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+websphere.java.security.exempt=true
+com.ibm.ws.logging.trace.specification=*=info:checkpoint=all
+io.openliberty.checkpoint.allowed.features=jaxb-2.2,xmlBinding-4.0
```

Comment:
Instead of allowed.features bootstrap property these public features should have `WLP-InstantOn-Enabled: true` header added.